src/main/java/com/ipt/kafkatopicupdates/producer/AuthorizationAvroProducer.java

```diff
@@ -36,7 +36,7 @@ private void scheduleFixedRateTask() {
         }
 
         ListenableFuture<SendResult<String, Authorization>> future =
-                authorizationKafkaTemplate.send(
+                (ListenableFuture<SendResult<String, Authorization>>) authorizationKafkaTemplate.send(
```

Comment:
Does this throw a warning in the IDE? My IDE says it's unnecessary.

Reply:
Good point. I think in an earlier version I needed to set this. Now it is not needed anymore.